types/compression-webpack-plugin/compression-webpack-plugin-tests.ts

```diff
@@ -47,7 +47,11 @@ const zlib: Configuration = {
                 windowBits: 20,
                 level: 7
             }
-        })
+        }),
+        new CompressionPlugin({
```

Comment:
`const zlib: Configuration = {`, seems like here they wanted to test `compressionOptions` argument, do you think this test should be moved higher for a better structure?

Reply:
Yes, it would look better, to be amended, thx!